src/sync/synchronizer.rs

```diff
@@ -413,7 +413,7 @@ impl Update {
 
         self.insert.push(insert);
         // also insert into map.
-        let inner_map = self.map.entry(outer_key.clone()).or_insert_with(HashMap::new);
+        let inner_map = self.map.entry(outer_key.clone()).or_default();
         inner_map.insert(inner_key, Value { type_id, data });
```

Comment:
Updated this line as per the suggestion provided in the failed build log.
![image](https://github.com/pravega/pravega-client-rust/assets/122247323/e14940cc-8011-4266-818b-4b65ee83b9ec)